source/tutorials/file-sets.md

```diff
@@ -23,23 +24,27 @@ Using these features directly can be tricky however:
   can address these problems.
   However, it's often hard to express the desired path selection using the `filter` function interface.
 
-In this tutorial you'll learn how to use the Nixpkgs [`fileset` library](https://nixos.org/manual/nixpkgs/unstable/#sec-fileset) to work with local files in derivations.
-It abstracts over built-in functionality and offers a safer and more convenient interface.
+In this tutorial you'll learn how to use Nixpkgs' [`lib.fileset` library](https://nixos.org/manual/nixpkgs/unstable/#sec-fileset) to work with local files in derivations.
+It abstracts over built-in functionality with a safer and more convenient interface.
```

Comment:
I'd revert both sentences. Apostrophising Nixpkgs looks really weird, and `abstracting over X with Y`, while grammatically correct can be misleading to be read as "built in functionality having a safer interface".